_pages/decide/task-flow-analysis.md

```diff
@@ -6,27 +6,31 @@ redirect_from: /task-flow-analysis/
 
 ## What it is
 
-A step-by-step analysis of a common task a user must perform that diagrams the various touch points and decision points a user goes through to accomplish the task. The touch points should be represented as steps taken by the user, as well as steps taken by the system.
+A step-by-step analysis of how a user will interact with a system in order to reach a goal. This analysis is documented in a diagram that traces a user's possible paths through sequences of tasks and decision points in pursuit of their goal. The tasks and decision points should represent steps taken by the user, as well as steps taken by the system.
 
 ## Reasons to use it
 
-To illustrate in a solution-agnostic way the overall flow that a user progresses through to accomplish a single task. Task flow analysis also demonstrates the relationship between tasks, and how they interconnect across a site.
+To validate a design team's understanding of users' goals, common scenarios, and tasks, and to illustrate in a solution-agnostic way the overall flow of tasks through which a user progresses to accomplish a goal.  Task flow diagrams also help surface obstacles in the way of users achieving their goal. 
```

Comment:
You have pinpointed my hidden agenda!  If they are based on user research and always verified by SMEs, then I see that as one of the many benefits of doing them. You document your understanding of the work, and then you take a representation of that understanding back to users and/or SMEs and confirm that that understanding is correct. (In my experience, walking through the flow with an SME or user in step 4 always results in modifications where my understanding was not quite right). 

....aaaand I just now see your comment re: step 4 :)